enumerable.rb

```diff
@@ -0,0 +1,24 @@
+require('./module')
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] `MyList` class implements `#each` method that yields all members of `@list` in succession. It means you should create a custom #each method inside `MyList` class that yields all members of @list one by each.